scenarios/scenario1/test/single.sh

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+
+start_dir=$( pwd )
+local_dir=$( cd $( dirname "$0" ); pwd )
+project_root=$( bash ${local_dir}/../../core/get_project_root.sh )
+echo "Project root: ${project_root}"
+
+
+scenario_dir=$(cd "${local_dir}/.."; pwd)
+echo "Scenario directory: ${scenario_dir}"
+cd "${scenario_dir}" || exit 1
+terraform init || exit $?
+
+ws_name="single"
+
+terraform workspace select ${ws_name} || terraform workspace new ${ws_name} || exit $?
+
+bastion_eip="80.158.3.174"
```

Comment:
this obsolete now

Reply:
this is required to retrieve public IPs in `post_build.sh`